src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/handlers/WarningCaseEventHandler.java

```diff
@@ -162,10 +165,10 @@ private void sendWarningMessage(String caseReference,
 
         warningMessageRequest.forEach(message -> {
                 if (message != null) {
+                    log.info("sendWarningMessage message:{}", message);
                     workflowApiClient.sendMessage(serviceAuthGenerator.generate(), message);
```

Comment:
What we really want to know is if we get an exception will it be set to Unprocessable or retried.  This logic is all from in DatabaseMessageConsumer.processMessage.  There is already logging in there but I think we can enhance it with some more details when it is considered Unprocessable.  E.g. output the CaseId, messageId and anything else.